ky-universal/node.js

```diff
@@ -1,4 +1,4 @@
-const fetch = require('node-fetch')
+const fetch = require('node-fetch/lib/index.js')
```

Comment:
This is causing to add node dependencies also to browser bundle

Browser bundle: https://unpkg.com/browse/node-fetch@2.6.0/browser.js (700B)
CJS: https://unpkg.com/browse/node-fetch@2.6.0/lib/index.js (41.3)

Can you please try this:

```suggestion
const fetch = process.server ? require('node-fetch/lib/index.js') : require('node-fetch/browser.js')

```

Reply:
Nice, that works!